src/components/Cards/CardTimeFrameFilter.js

```diff
@@ -0,0 +1,20 @@
+import ClassNames from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const CardTimeFrameFilter = ({ children, className, ...props }) => {
+  const classes = ClassNames('dropdown card-pf-time-frame-filter', className);
```

Comment:
This is another case of a custom button group it seems... we just want to remove the `bsClass` (should not have to render a `div`, let the `ButtonGroup` do it for you): 
```
 const CustomButtonGroup = props => {
    return <ButtonGroup {...props} bsClass=" " />;
  };
```

Reply:
(full example from before was here: https://github.com/patternfly/patternfly-react/blob/master/src/components/Table/TableDropdownKebab.js )